### PR TITLE
Fix: screen reader support and focus indication on links

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview

- Added aria-live tag on the status message, to enable the screen reader to read the message.

- Added an aria-label for the dismiss button on the status message and the terms of use message bar .

- Enabled multiline functionality on the terms of use message bar for better focus on the links when using keyboard navigation.

### Demo

image

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

To test the status message accessibility
Ensure a screen reader is activated (e.g Narrator)

Step 1: Launch the application on your browser.
Step 2: Navigate to “Run Query” buttons using tab key and activate it by pressing enter key.
Step 3: Observe screen reader is narrating “Status” message.

To test the focus indication on the links
Step 1: Launch the application on your browser.​
Step 2: Navigate to 'Microsoft APIs Terms of Use' and 'Microsoft Privacy Statement'  link by using tab key.
Step 3: Observe that a focus indicator is clearly visible.

### CODE REVIEW

File: src/app/views/app-sections/StatusMessages.tsx [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+  isMultiline={false} onDismiss={actions.clearQueryStatus}
+  dismissButtonAriaLabel='Close' aria-live={'assertive'}>
```
Comment (jobala) [inline src/app/views/app-sections/StatusMessages.tsx:8]: You can improve readability by arranging the props as shown below.
```
<MessageBar
        messageBarType={}
        isMultiline={}
        onDismiss={}
>
```

Comment (ElinorW) [reply src/app/views/app-sections/StatusMessages.tsx:8]: Oh yeah, thank you for this

File: src/app/views/app-sections/StatusMessages.tsx [hunk 2]
```diff
@@ -4,7 +4,10 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+  isMultiline={false}
```
Comment (jobala) [inline src/app/views/app-sections/StatusMessages.tsx:7]: Indent by two spaces  :wink:

Comment (Kachulio1) [reply src/app/views/app-sections/StatusMessages.tsx:7]: I think we should add a pre-commit hook that runs prettier no? 🤔

Comment (jobala) [reply src/app/views/app-sections/StatusMessages.tsx:7]: Yes, we should.

File: src/app/views/app-sections/StatusMessages.tsx [hunk 3]
```diff
@@ -4,7 +4,10 @@ import { FormattedMessage } from 'react-intl';
 export function statusMessages(queryState: any, actions: any) {
   return queryState && (
   <MessageBar messageBarType={queryState.messageType}
-  isMultiline={false} onDismiss={actions.clearQueryStatus}>
+    isMultiline={false}
+    onDismiss={actions.clearQueryStatus}
+    dismissButtonAriaLabel='Close'
```
Comment (ddyett) [inline src/app/views/app-sections/StatusMessages.tsx:9]: how does a screen handle this on a localized page?

Comment (ElinorW) [reply src/app/views/app-sections/StatusMessages.tsx:9]: The screen reader is able to read it as well on a localized page

Comment (ddyett) [reply src/app/views/app-sections/StatusMessages.tsx:9]: yes but does it read the english word or is it localized.

Comment (ElinorW) [reply src/app/views/app-sections/StatusMessages.tsx:9]: It's localized

**Pull request merged**